charts/sequencer/files/cometbft/config/genesis.json

```diff
@@ -119,13 +119,31 @@
       {{- if $index }},{{- end }}
       {{ include "sequencer.address" $value }}
       {{- end }}
-    ]
+    ],
+    "connect": {
+      "marketMap": {
+        "marketMap": {
+          "markets": {}
+        },
+        "params": {
+          "marketAuthorities": [],
+          "admin": "{{ .Values.genesis.marketAdminAddress }}"
+        }
+      },
+      "oracle": {
+        "currencyPairGenesis": [],
+        "nextId": "0"
+      }
+    }
     {{- if not .Values.global.dev }}
     {{- else }}
     {{- end}}
   },
   "chain_id": "{{ .Values.genesis.chainId }}",
   "consensus_params": {
+    "abci": {
```

Comment:
nit: should move under else statement, marking breaking changes. You will have to add the closure

Reply:
is this breaking since it's not adding a new value? what's the correct syntax for this? do i move the whole `consensus_params` block into the else statement?